flink-connectors/flink-connector-hbase-1.4/src/main/java/org/apache/flink/connector/hbase1/HBase1DynamicTableFactory.java

```diff
@@ -65,16 +65,23 @@
     @Override
     public DynamicTableSource createDynamicTableSource(Context context) {
         TableFactoryHelper helper = createTableFactoryHelper(this, context);
+        helper.forwardOptions(
+                TABLE_NAME,
+                ZOOKEEPER_ZNODE_PARENT,
+                ZOOKEEPER_QUORUM,
+                NULL_STRING_LITERAL,
+                LOOKUP_ASYNC,
```

Comment:
not an Hbase1 option? at least is is neither among optional and required options

Reply:
Seems like it's an hbase 1 option as well, as it's used in `HBaseConnectorOptionsUtil#getHBaseLookupOptions`, used by the hbase 1 connector as well. Will add among optional options, as hbase 2